### PR TITLE
fix: Fix clippy errors

### PR DESCRIPTION
**Pull request by morgsmccauley**

### CODE REVIEW

File: database/src/adapters/blocks.rs [hunk 1]
```diff
@@ -43,10 +43,10 @@ pub async fn get_latest_block_before_timestamp(
     pool: &actix_diesel::Database<PgConnection>,
     timestamp: u64,
 ) -> anyhow::Result<models::Block> {
-    Ok(schema::blocks::table
```
Comment (morgsmccauley) [inline database/src/adapters/blocks.rs:46]: https://rust-lang.github.io/rust-clippy/master/index.html#needless_question_mark

File: database/src/adapters/access_keys.rs [hunk 2]
```diff
@@ -34,7 +34,7 @@ pub async fn handle_access_keys(
                         (public_key.to_string(), account_id.to_string()),
                         models::access_keys::AccessKey::from_action_view(
                             public_key,
-                            &account_id,
+                            account_id,
```
Comment (morgsmccauley) [inline database/src/adapters/access_keys.rs:37]: https://rust-lang.github.io/rust-clippy/master/index.html#needless_borrow

File: circulating-supply/src/main.rs [hunk 3]
```diff
@@ -216,7 +216,7 @@ async fn compute_circulating_supply_for_block(
 
     Ok(models::aggregated::circulating_supply::CirculatingSupply {
         computed_at_block_timestamp: BigDecimal::from(block_timestamp),
-        computed_at_block_hash: (&block.block_hash).to_string(),
+        computed_at_block_hash: block.block_hash.to_string(),
```
Comment (morgsmccauley) [inline circulating-supply/src/main.rs:219]: https://rust-lang.github.io/rust-clippy/master/index.html#needless_borrow

File: database/src/retryable.rs [hunk 4]
```diff
@@ -2,7 +2,7 @@
 macro_rules! await_retry_or_panic {
     ($query: expr, $number_of_retries: expr, $error_message: expr, $debug_structs: expr $(, $is_error_handled:expr)? $(,)?) => {
         {
-            let mut interval = crate::INTERVAL;
+            let mut interval = $crate::INTERVAL;
```
Comment (morgsmccauley) [inline database/src/retryable.rs:5]: https://rust-lang.github.io/rust-clippy/master/index.html#crate_in_macro_def

File: circulating-supply/src/lockup_types.rs [hunk 5]
```diff
@@ -98,7 +98,7 @@ pub enum TransfersInformation {
 
 /// Describes the status of transactions with the staking pool contract or terminated unvesting
 /// amount withdrawal.
-#[derive(BorshDeserialize, BorshSerialize, Deserialize, Serialize, PartialEq)]
+#[derive(BorshDeserialize, BorshSerialize, Deserialize, Serialize, PartialEq, Eq)]
```
Comment (morgsmccauley) [inline circulating-supply/src/lockup_types.rs:101]: https://rust-lang.github.io/rust-clippy/master/index.html#derive_partial_eq_without_eq

**Review by khorolets — APPROVED** [review]

**Review by telezhnaya — APPROVED** [review]

**Pull request merged**